python/src/aiconfig/editor/client/src/components/prompt/prompt_input/PromptInputConfigRenderer.tsx

```diff
@@ -11,10 +12,12 @@ export default memo(function PromptInputConfigRenderer({
   input,
   onChangeInput,
 }: Props) {
+  const {readOnly} = useContext(AIConfigContext);
   return (
     <Textarea
       value={input as string}
       onChange={(e) => onChangeInput(e.target.value)}
+      disabled={readOnly}
```

Comment:
Instead of disabled textarea, let's render the nicer TextRenderer that text outputs use. Maybe we could also wrap that with a Spoiler in case the input is long (see TextInputCellRenderer in lastmile - https://github.com/lastmile-ai/lastmile/blob/bd3b54248682aed2720f7c2c3caefc8ebda335ee/src/components/playground_v2/cell_renderers/TextInputCellRenderer.tsx#L143-L156)

Reply:
updated 
readOnly = False
<img width="1055" alt="Screenshot 2024-01-19 at 10 16 06 AM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/c662e114-8c7c-488f-89e4-ad9e9ac6b460">

readOnly = True
<img width="995" alt="Screenshot 2024-01-19 at 10 17 10 AM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/110306a1-ec08-4836-a732-186a5e3a91e5">